src/data/support-title.csv

```diff
@@ -62,7 +62,7 @@
 "[押して忍べど燃ゆるもの]","[Unrestrainably Burning With Passion]"
 "[夢はホントに叶うんだ！]","[Dreams Do Come True!]"
 "[今ぞ盛りのさくら花]","[Sakura in Full Bloom]"
-"[花嫁たるもの！！]",""
+"[花嫁たるもの！！]","[I'm a bride!]"
 "[召しませふぁーすとBite！]",""
-"[おもい、ねがう]",""
+"[おもい、ねがう]","[I Think and I Hope]"
```

Comment:
omoi here should be thought, or feeling. while negau kinda like wish. [A Feeling, a Wish]

Reply:
Yeah, i was confused with the translated too